x/auth/client/testutil/suite.go

```diff
@@ -1604,6 +1608,7 @@ func (s *IntegrationTestSuite) TestAuxSigner() {
 	}
 }
 
+// TODO to re-enable in #12274
```

Comment:
```suggestion
```

This should not be re-enabled.

Reply:
why are we keeping tests that just get skipped?